backend/users/models.py

```diff
@@ -4,7 +4,11 @@
 User = get_user_model()
 
 
-# Create your models here.
+class LoginGovUser(models.Model):
+    user = models.OneToOneField(User, on_delete=models.CASCADE)
+    login_id = models.CharField(max_length=100, unique=True)
+
+
 class UserProfile(models.Model):
     user = models.OneToOneField(User, related_name="profile", on_delete=models.CASCADE)
```

Comment:
[NB] Same here about cascading as above for line 8

Reply:
Same reply as above